src/components/Overview/Overview.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { getFormData } from '../../selectors';
+import { FormLabel, Headline, Typography } from '@vfde-react/pattern';
+import './Overview.css';
+
+const Overview: React.FC = () => {
+  const formData = useSelector(getFormData);
+  return (
+    <>
+      <Headline
+        children="Overview Page"
+        level={2}
+        mobileAlign="left"
+        align="left"
```

Comment:
same comment for align and mobileAlign props

Reply:
this is because of the warnings appeared from the component i was trying to reduce the number of warnings by defining the data